recipes/quippy/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "quippy" %}
+{% set version = "0.0.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/libAtoms/QUIP/archive/public.tar.gz
+  sha256: 1f35a21b7fea2ec55dfb0fb6f7d73523e8151386748f3ed0e658a77d619535bd
+  patches:
+    - options.patch
+    - fox.patch
+    - quippy.patch
```

Comment:
For you own future self sake, add some comments on what these patches do :smile:

Reply:
I added the following comments:
```
- options.patch  # configure quip by patching the makefile rather than using the interactive configuration 
- fox.patch  # use the fox package from conda-forge
- quippy.patch  # use the f90wrap package from conda-forge 
```
Basically `fox.patch` and `quippy.patch` are required to use the corresponding conda-forge packages rather than downloading the repositories and compiling the dependencies again. And `options.patch` is used to bypass `make config` which requires an interactive shell.